src/js/base/module/Codeview.js

```diff
@@ -16,11 +16,17 @@ export default class CodeView {
     this.$editable = context.layoutInfo.editable;
     this.$codable = context.layoutInfo.codable;
     this.options = context.options;
+    this.CodeMirrorConstructor = CodeMirror; 
```

Comment:
It looks like we don't need to keep `env.hasCodeMirror` when this patch is applied.
How about to remove `env.hasCodeMirror` from `core/env.js` and https://github.com/summernote/summernote/blob/83fb867d36501296591ddabc9400a4eb1297effa/src/js/base/module/Codeview.js#L5-L8 ?

Reply:
good idea.